README.md

```diff
@@ -71,7 +71,22 @@ Not yet, but flake.lock makes it reproducible.
 
 ###	Can I use it with trusty old `nix-shell`?
 
-Probably, but I haven't put time into how.  I'm leaning into flakes.
+Absolutely! The `shell.nix` provides a flakes-compatible shell that works with `nix-shell`. It selects the `application` shell by default, but you can be specific about it. E.g.
```

Comment:
I wondered whether there should be a default devShell as well.

Reply:
I guess it's a matter of deciding what "default" means :)

Reply:
Yep, exactly.  I mostly envision this being embedded in projects with direnv, but what we have already is a nice one-liner for unnixxed projects.